src/build-scripts/ci-build.bash

```diff
@@ -27,7 +27,8 @@ if [[ ${GITHUB_ACTIONS} == true ]] ; then
     OIIO_CMAKE_FLAGS+=" -DCMAKE_UNITY_BUILD=${CMAKE_UNITY_BUILD:=ON} -DCMAKE_UNITY_BUILD_MODE=${CMAKE_UNITY_BUILD_MODE:=BATCH}"
 fi
 
-cmake -S . -B build -G "$CMAKE_GENERATOR" \
+pushd build
+cmake .. -G "$CMAKE_GENERATOR" \
```

Comment:
I think we regressed here. Now that cmake 3.15 is the minimum we support, we can use -B instead of having to cd into a build directory.